### PR TITLE
Allows boundary conditions to be set in KPP and ModularKPP from Model…

### PR DESCRIPTION
**Pull request by glwagner**
… constructor

### CODE REVIEW

**Comment by glwagner** [comment]
cc @sandreza

File: src/models/k_profile_parameterization.jl [hunk 1]
```diff
@@ -73,25 +74,25 @@ mutable struct Model{S, G, T, U, B} <: AbstractModel{S, G, T}
     state       :: State{T}
 end
 
+function BoundaryConditions(FT=Float64; U = DefaultBoundaryConditions(FT),
```
Comment (sandreza) [inline src/models/k_profile_parameterization.jl]: Can you write a docstring? How would one set boundary conditions using this function?

**Review by sandreza — CHANGES_REQUESTED** [review]
Just need a docstring, then ready to merge.

File: src/models/k_profile_parameterization.jl (hunk 1 above)
Comment (glwagner) [reply src/models/k_profile_parameterization.jl]: Done.

**Comment by sandreza** [comment]
Just checked it. Works well!

**Pull request merged**